templates/telemetry/pioneer-study/pioneer-study.1.schema.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
There also needs to be study specify schemas for validation after decryption.
i.e. templates/pioneer-study/foobar.1.schema.json